### PR TITLE
I nominate Smort

### PR DESCRIPTION
**Pull request by Michael-Loula**
I have learned to yolo swag and I will value that forever 🙏

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ const hofer_str = `
 * @Jingkai
 * @Russell
 * @annnn
+* @Smort
```
Comment (smortime) [inline main.go:20]: is this thread safe?

**Review by smortime — APPROVED** [review]
LGTM 🌴 

I would like to thank the fam for giving me the opportunity to achieve my dreams.

File: main.go (hunk 1 above)
Comment (Michael-Loula) [reply main.go:20]: I think only smort_r is thread safe

Comment (Michael-Loula) [reply main.go:20]: bbut yolo

**Pull request merged**